specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3735,9 +3739,64 @@
         "destinationInvoiceSectionId": {
           "type": "string",
           "description": "The destination invoice section id."
+        },
+        "destinationBillingProfileId": {
+          "type": "string",
+          "description": "The destination billing profile id."
+        }
+      }
+    },
+    "ValidateSubscriptionTransferEligibilityResult": {
```

Comment:
These new types aren't used anywhere in your APIs. What was your intention with these types?

Reply:
Fixed it, sorry forgot to copy the eligibility APIs from our local json files